src/pages/SearchPage.js

```diff
@@ -78,8 +83,18 @@ class SearchPage extends Component {
         Timing.end(CONST.TIMING.SEARCH_RENDER);
     }
 
+    componentDidUpdate(prevProps) {
+        if (this.props.reports === prevProps.reports) {
```

Comment:
Maybe `_.isEqual()` would be better here since I think maybe this is just checking for same object reference and not object equality.